spec/models/post_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Se eu não me engano, é recomendado remover esses arquivos automáticos quando a gente não for usar